build-tools/automation/yaml-templates/setup-test-environment.yaml

```diff
@@ -55,3 +55,10 @@ steps:
     solution: build-tools/xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks.csproj
     configuration: ${{ parameters.configuration }}
     msbuildArguments: /restore /bl:$(System.DefaultWorkingDirectory)/bin/Test${{ parameters.configuration }}/BootstrapTasks.binlog
+
+- task: MSBuild@1
+  displayName: build check-boot-times.csproj
```

Comment:
Can we remove this and replace it with a command that will build and run to the main YAML file, as part of the apk test job? We could likely do this with one or two new msbuild invocations here - https://github.com/xamarin/xamarin-android/blob/master/build-tools/automation/azure-pipelines.yaml#L624

I would prefer to not build this assembly for every test job, and would rather not run `RunAndroidEmulatorCheckBootTimes` as part of every `AcquireAndroidTarget` call.